cmd/internal/find/primary/meta.go

```diff
@@ -56,4 +59,22 @@ N                   => \d+ (i.e. some number > 0)
 var metaPrimary = Parser.add(&primary{
     tokens: []string{"-meta", "-m"},
     parseFunc: meta.Parse,
+    optionsSetter: func(opts *types.Options) {
```

Comment:
This capability will be useful for other primaries as well, like e.g. `delete` where for `delete`, the `depth` option must be set (otherwise, the parent would be deleted before the children).